server/server.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const app = express();
+const bodyParser = require('body-parser');
+const path = require('path');
+const cors = require('cors');
+
+app.use(express.static(path.join(__dirname, '../public/')));
+
+app.use(cors());
+
+app.get('/', (req, res) => {
```

Comment:
Here is where you will want to link up your route to be forwarded to your service server and then respond to the client in the same way you would on your service. It's all pretty much the same as your service server, just in a different environment.